From c7572d7660140a3f61c922cde11be823dfa58a32 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 14 Feb 2013 14:26:06 +0100
Subject: [PATCH 1/2] Refer to the dpkg man pages for programs not described in
 the appendices

This will allow to progressively remove specific program sections, that
have been either integrated into the policy or the dpkg documentation.
---
 policy.sgml | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index ddf7698..31d7232 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -10846,10 +10846,7 @@ END-INFO-DIR-ENTRY
 
       <p>
 	The utility programs which are provided with <prgn>dpkg</prgn>
-	for managing various system configuration and similar issues,
-	such as <prgn>update-rc.d</prgn> and
-	<prgn>install-info</prgn>, are not described in detail here -
-	please see their man pages.
+	not described in detail here, are documented in their man pages.
       </p>
 
       <p>
-- 
1.8.1.3

